index.html

```diff
@@ -48,8 +48,13 @@
 
     <section id="hackathon" class="content-section text-center">
       <div class="row">
-        <div class="col-lg-8 col-lg-offset-2">
-          <h2>Hacakthon</h2>
+        <div class="col-lg-8 col-lg-offset-2" style= color:black; >
+          <h2 style= color:#4B0583; text-style:bold; >Exploring the Tech Space Together</h2>
+             <p>Spectra is the first all­ ladies hackathon in the west coast taking place in the Bay Area! This is a unique event to bring together high schoolers, college students, and women in technology from all over.</p>
```

Comment:
Please align the indents correctly.
h2 should be aligned with the p tags. Each tab is 2 spaces.